rust/migrate-wicked/tests/vlan/wicked_xml/vlan.xml

```diff
@@ -0,0 +1,30 @@
+<interface origin="compat:suse:/etc/sysconfig/network/ifcfg-eth0.10">
+  <name>eth0.10</name>
+  <control>
+    <mode>boot</mode>
+  </control>
+  <firewall>
+    <zone>trusted</zone>
+  </firewall>
```

Comment:
Just wondering, is this required for vlans? Because otherwise I would remove it, because this might cause issues when we implement firewall.

Reply:
I guess it's not